### ISSUE TITLE
Support to armv7 on iOS

### ISSUE THREAD

**Issue by gobetti** [JetBrains/kotlin-native#2185]
**TL;DR:**
What do I need to do to support armv7 on iOS? Or is there currently no support for that architecture?

**Detailed:**
The [release notes](https://github.com/JetBrains/kotlin-native/blob/master/RELEASE_NOTES.md) mention support to armv7 on iOS devices - it originally said only arm64 but then armv7 was added to the notes in https://github.com/JetBrains/kotlin-native/commit/e2b051e2d5e857e8305d149f85e3fe588a32b263

I found other issues discussing about armv7 support (#1178, #1460) but I couldn't find an official request nor a pull request clearly adding support, as it seems to have been part of v0.8 but there's no mention to it in [its changelog](https://github.com/JetBrains/kotlin-native/releases/tag/v0.8).

I locally updated the [calculator sample](https://github.com/JetBrains/kotlin-native/tree/master/samples/calculator) to konan 0.9.2, it's working fine for simulators (x64) and newer devices (arm64), however even if I add armv7 to the supported architectures and modify the `KONAN_TASK` build setting to armv7, I can't build it for an iPhone 4S.

```
* What went wrong:
Task 'compileKonanKotlinArithmeticParserIos_armv7' not found in project ':ios'. Some candidates are: 'compileKonanKotlinArithmeticParserIos_arm64'.
```

Do I need any other changes to support armv7?

Thank you very much!

**Comment by thevery** [comment]
you must add `ios_arm32` target here:
https://github.com/JetBrains/kotlin-native/blob/master/samples/calculator/ios/build.gradle#L14

**Comment by gobetti** [comment]
Hi @thevery , thanks a lot for your very quick answer!
Adding `ios_arm32` allows me to run the compileKonan task specifying `arm32` and it builds fine for `armv7`. That's pretty enough to close this issue, but I'd like to open a pull request updating the calculator sample and in order to do that, I'll need to update the `KONAN_TASK` value in the pbxproj.

Currently that value has `arm64` hardcoded (x64 if simulator). Do you have a suggestion to have this value dynamic according to the target device? Maybe if it was called `armv7` and `armv7s` this would be simpler.

**Comment by thevery** [comment]
Calculator sample is probably outdated anyway, see https://github.com/h0tk3y/k-new-mpp-samples and https://github.com/JetBrains/kotlin-mpp-example
